frontend/src/core/editor/selectors.js

```diff
@@ -73,8 +73,17 @@ export function _existingTranslation(
                     fluent.getReconstructedMessage(entity.original, translation)
                 );
                 existingTranslation = history.translations.find(
-                    t => t.string === reconstructed
-                )
+                    t => {
+                        // Some Fluent strings are missing the trailing newline
+                        // character while stored in the database. In order for this
+                        // comparison to work, we need to make sure it is there.
+                        let str = t.string;
+                        if (!str.endsWith('\n')) {
+                            str += '\n';
+                        }
+                        return str === reconstructed;
+                    }
+                );
```

Comment:
Why not instead parse & serialize as we do in line 52?

I'd also file a bug to investigate why the strings are stored inconsistently and reference it here.

Reply:
Because it's a lot more work, for just this simple fix.

Reply:
OK. I'd still reference the bug you filed here (and/or vice-versa).